src/headless/shared/connection.js

```diff
@@ -247,6 +247,12 @@ export class Connection extends Strophe.Connection {
      * @param { String } message
      */
     onConnectStatusChanged (status, message) {
+        /**
+         * Synchronous event triggered on connection status change
+         * @event _converse#onConnectStatusChanged
+         */
+        api.trigger('connectionStatusChange', { status, message });
```

Comment:
Thanks, but the JSDoc description is wrong, this is not a synchronous event and the event name in the JSDoc is `onConnectStatusChanged` while the actual event name is `connectionStatusChange`.